README.md

```diff
@@ -4,16 +4,42 @@ Language server for Ralph.
 
 Currently supports text document level events and diagnostics.
 
+# Installation
```

Comment:
Please consider moving this after `Install VSCode plugin` and `Install Neovim plugin`. Most devs probably just want to download plugins for their IDEs.